facade/nodeFacade_test.go

```diff
@@ -1747,7 +1924,248 @@ func TestNodeFacade_InternalValidatorsInfo(t *testing.T) {
+			return providedResponse, nil
+		},
+	}
+	nf, _ := NewNodeFacade(args)
+
+	response, err := nf.GetEpochStartDataAPI(0)
+	require.NoError(t, err)
+	require.Equal(t, providedResponse, response)
+}
+
+func TestNodeFacade_GetConnectedPeersRatings(t *testing.T) {
+	t.Parallel()
+
+	providedResponse := "ratings"
+	args := createMockArguments()
+	args.Node = &mock.NodeStub{
+		GetConnectedPeersRatingsCalled: func() string {
+			return providedResponse
+		},
+	}
+	nf, _ := NewNodeFacade(args)
+
+	response := nf.GetConnectedPeersRatings()
+	require.Equal(t, providedResponse, response)
+}
+
+func TestNodeFacade_GetBlockByHash(t *testing.T) {
+	t.Parallel()
+
+	providedResponse := &api.Block{
+		Nonce: 123,
+		Round: 321,
+	}
+	args := createMockArguments()
+	args.ApiResolver = &mock.ApiResolverStub{
+		GetBlockByHashCalled: func(hash string, options api.BlockQueryOptions) (*api.Block, error) {
+			return providedResponse, nil
+		},
+	}
+	nf, _ := NewNodeFacade(args)
+
+	response, err := nf.GetBlockByHash("hash", api.BlockQueryOptions{})
+	require.NoError(t, err)
+	require.Equal(t, providedResponse, response)
+}
+
+func TestNodeFacade_GetBlockByNonce(t *testing.T) {
+	t.Parallel()
+
+	providedResponse := &api.Block{
+		Nonce: 123,
+		Round: 321,
+	}
+	args := createMockArguments()
+	args.ApiResolver = &mock.ApiResolverStub{
+		GetBlockByNonceCalled: func(nonce uint64, options api.BlockQueryOptions) (*api.Block, error) {
+			return providedResponse, nil
+		},
+	}
+	nf, _ := NewNodeFacade(args)
+
+	response, err := nf.GetBlockByNonce(0, api.BlockQueryOptions{})
+	require.NoError(t, err)
+	require.Equal(t, providedResponse, response)
+}
+
+func TestNodeFacade_GetAlteredAccountsForBlock(t *testing.T) {
+	t.Parallel()
+
+	providedResponse := []*outport.AlteredAccount{
+		{
+			Nonce:   123,
+			Address: "address",
+		},
+	}
+	args := createMockArguments()
+	args.ApiResolver = &mock.ApiResolverStub{
+		GetAlteredAccountsForBlockCalled: func(options api.GetAlteredAccountsForBlockOptions) ([]*outport.AlteredAccount, error) {
+			return providedResponse, nil
+		},
+	}
+	nf, _ := NewNodeFacade(args)
+
+	response, err := nf.GetAlteredAccountsForBlock(api.GetAlteredAccountsForBlockOptions{})
+	require.NoError(t, err)
+	require.Equal(t, providedResponse, response)
+}
+
+func TestNodeFacade_GetInternalStartOfEpochMetaBlock(t *testing.T) {
+	t.Parallel()
+
+	providedResponse := "meta block"
+	args := createMockArguments()
+	args.ApiResolver = &mock.ApiResolverStub{
+		GetInternalStartOfEpochMetaBlockCalled: func(format common.ApiOutputFormat, epoch uint32) (interface{}, error) {
+			return providedResponse, nil
+		},
+	}
+	nf, _ := NewNodeFacade(args)
+
+	response, err := nf.GetInternalStartOfEpochMetaBlock(0, 0)
+	require.NoError(t, err)
+	require.Equal(t, providedResponse, response)
+}
+
+func TestNodeFacade_Close(t *testing.T) {
+	t.Parallel()
+
+	nf, _ := NewNodeFacade(createMockArguments())
+	require.NoError(t, nf.Close())
```

Comment:
it seems like the context created on the constructor isn't used anywhere. might remove it and the cancel function

Reply:
indeed, fixed